src/renderer/components/wallet/Settings.tsx

```diff
@@ -64,111 +45,137 @@ const Settings: React.FC<Props> = (props: Props): JSX.Element => {
         userAccounts,
         O.map((accounts) => (
           <Col key={'accounts'} sm={{ span: 24 }} md={{ span: 12 }}>
-            <StyledSubtitle>{intl.formatMessage({ id: 'setting.account.management' })}</StyledSubtitle>
-            <StyledAccountCard>
+            <Styled.Subtitle>{intl.formatMessage({ id: 'setting.account.management' })}</Styled.Subtitle>
+            <Styled.AccountCard>
               <List
                 dataSource={accounts}
                 renderItem={(item, i: number) => (
-                  <StyledListItem key={i}>
-                    <StyledChainName>{item.chainName}</StyledChainName>
+                  <Styled.ListItem key={i}>
+                    <Styled.ChainName>{item.chainName}</Styled.ChainName>
                     {item.accounts.map((acc, j) => (
-                      <StyledChainContent key={j}>
-                        <StyledAccountPlaceholder>{acc.name}</StyledAccountPlaceholder>
-                        <StyledAccountContent>
-                          <StyledAccountAddress>{acc.address}</StyledAccountAddress>
+                      <Styled.ChainContent key={j}>
+                        <Styled.AccountPlaceholder>{acc.name}</Styled.AccountPlaceholder>
+                        <Styled.AccountContent>
+                          <Styled.AccountAddress>{acc.address}</Styled.AccountAddress>
                           {acc.type === 'external' && (
                             <Button type="link" danger>
                               <StopOutlined />
                             </Button>
                           )}
-                        </StyledAccountContent>
-                      </StyledChainContent>
+                        </Styled.AccountContent>
+                      </Styled.ChainContent>
                     ))}
-                  </StyledListItem>
+                  </Styled.ListItem>
                 )}
               />
-            </StyledAccountCard>
+            </Styled.AccountCard>
           </Col>
         )),
         O.getOrElse(() => <></>)
       ),
     [intl, userAccounts]
   )
 
+  const changeNetworkHandler: MenuProps['onClick'] = useCallback(
+    (param) => {
+      console.log('changeNetworkHandler:', param)
+      const asset = param.key as Network
+      changeNetwork(asset)
+    },
+    [changeNetwork]
+  )
+
+  const networkMenu = useMemo(
+    () => (
+      <Styled.NetworkMenu onClick={changeNetworkHandler}>
+        {(['testnet', 'chaosnet', 'mainnet'] as Network[]).map((network) => (
```

Comment:
imo its might be useful to move this array to app-level constants. what do u think ? if we create constant like
```ts
const availableNetwork = ['testnet', 'chaosnet', 'mainnet']
```
then we can create dynamic type for `Network` like
```ts
type Network = typeof availableNetwork[0]
```

Reply:
> to move this array to app-level constants.

Good idea! 

In case of:
```
const availableNetwork = ['testnet', 'chaosnet', 'mainnet']
type Network = typeof availableNetwork[0]
```

`Network` can be any string then. That's why I prefer to keep it as before 
```
type Network = 'testnet' | 'chaosnet' | 'mainnet'`
```
to be more type-safety.

Reply:
yes. my bad. for some reason i had a better opinion about TS :(